app/controllers/orgs_controller.rb

```diff
@@ -66,7 +75,7 @@ def set_org
     def org_params
       params.
         require(:org).
-        permit(:name, :description, :url, :contact_id,
+        permit(:name, :description, :url, :contact_name,
```

Comment:
Good catch for changing all fields (contact / org / etc.) to name instead of id!